app/uk/gov/hmrc/gform/upscan/UpscanController.scala

```diff
@@ -114,8 +114,7 @@ class UpscanController(
                       )
                     case ConfirmationFailure.UpscanFailure(FailureDetails("REJECTED", _)) =>
                       mkFlash(
-                        "file.error.type",
```

Comment:
`file.error.type` is message we don't like and thus should be removed from messages.... also there are still two places to refer to it in the code.

Reply:
fixed, I left it with the same key but changed to the same content as file.error.rejected